packages/react/figma.config.json

```
@@ -0,0 +1,10 @@
+{
+  "codeConnect": {
+    "parser": "react",
+    "importPaths": {
+      "src/*": "@primer/react-brand"
+    },
+    "include": ["src/**/*.figma.tsx", "src/**/*.tsx"],
+    "exclude": ["src/**/*.stories.tsx", "src/**/*.test.tsx"]
```

```suggestion
    "include": ["src/**/*.figma.tsx"]
```

I guess this could be simplified to just include `*.figma.tsx`, right?